python/pipeline/vreso.py

```diff
@@ -0,0 +1,175 @@
+import scanreader
+import numpy as np
+import datajoint as dj
+from pipeline.exceptions import PipelineException
+from pipeline import reso, experiment, mice, notify, shared
+from pipeline.utils import galvo_corrections, signal, quality, mask_classification, performance, h5
```

Comment:
a lot of these are not used in the code, importing them makes people believe they are

Reply:
Since this schema a rough draft, these libraries will be used later. I removed parts that are incomplete.